ros_ws/src/simulation/vesc_sim/include/vesc_sim.h

```diff
@@ -21,6 +23,7 @@ class VESCSimulator
     private:
     ros::NodeHandle m_node_handle;
     ros::Publisher m_odometry_publisher;
+    boost::shared_ptr<tf::TransformBroadcaster> m_tf_publisher;
```

Comment:
Is there a reason why you are using a `shared_ptr`? As far as I can tell the `shared_ptr` does not seem necessary (as the object is not shared between other objects or functions).

Reply:
This code was... uh... _inspired_ by the vesc_to_odom node in the vesc_ackermann package. There is no reason why a `shared_ptr` is used here, I just know that it works. If you want to make some improvements, feel free to push anything onto the branch

Reply:
I removed the `shared_ptr` and it seems to work, but please tests it also on your computer as I'd like to prevent "but it worked on my computer!!11"-Problems ^^,

Reply:
It seems to work just fine 👍